.github/workflows/validate-clean-linux-binaries.yml

```diff
@@ -0,0 +1,248 @@
+      PYTHON_VERSION: ${{ matrix.python_version }}
+    steps:
+      - name: Checkout PyTorch builder
+        uses: actions/checkout@v2
+      - name: Install conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.python_version }}
+          auto-update-conda: true
+          miniconda-version: "latest"
+          activate-environment: testenv
+      - name: Check nvidia smi
+        run: |
+          nvidia-smi
+      - name: Install pytorch and smoke test
+        env:
+          ENV_NAME: env-conda-${{ github.run_id }}
+          CUDA_VER: ${{ matrix.cuda_version }}
+          CHANNEL: ${{matrix.conda_channel}}
+        run: |
+          set -ex
+          conda create -yp ${ENV_NAME} python=${PYTHON_VERSION} numpy
+          case $CUDA_VER in
+            ("10.2") { conda install -p ${ENV_NAME} pytorch torchvision torchaudio cudatoolkit=${CUDA_VER} -c ${CHANNEL};
+                      conda run -p ${ENV_NAME} python3 ./test/smoke_test/smoke_test.py;}
+                     ;;
+            ("11.3") { conda install -p ${ENV_NAME} pytorch torchvision torchaudio cudatoolkit=${CUDA_VER} -c ${CHANNEL};
+                      conda run -p ${ENV_NAME} python3 ./test/smoke_test/smoke_test.py; }
+                     ;;
+            ("11.6") { conda install -p ${ENV_NAME} pytorch torchvision torchaudio cudatoolkit=${CUDA_VER} -c ${CHANNEL} -c conda-forge;
+                      conda run -p ${ENV_NAME} python3 ./test/smoke_test/smoke_test.py; }
+                     ;;
+            ("11.7") { conda install pytorch torchvision torchaudio pytorch-cuda=11.7 -c pytorch-nightly -c nvidia;
+                      conda run -p ${ENV_NAME} python3 ./test/smoke_test/smoke_test.py; }
+                     ;;
+           (*) echo "No binary available"
+                 ;;
+          esac
+          
+  validate-linux-conda-cpu-binaries:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 360
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version: ["3.7", "3.8", "3.9", "3.10"]
+        conda_channel: [pytorch-nightly, pytorch-test, pytorch]
+    env:
+      PYTHON_VERSION: ${{ matrix.python_version }}
+    steps:
+      - name: Checkout PyTorch builder
+        uses: actions/checkout@v2
+      - name: Install conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.python_version }}
+          auto-update-conda: true
+          miniconda-version: "latest"
+          activate-environment: testenv
+      - name: Install pytorch and smoke test
+        env:
+          ENV_NAME: env-conda-${{ github.run_id }}
+          CHANNEL: ${{matrix.conda_channel}}
+        run: |
+          set -ex
+          conda create -yp ${ENV_NAME} python=${PYTHON_VERSION} numpy
+          conda install pytorch torchvision torchaudio cpuonly -c pytorch;
+          conda run --cwd /tmp -p ${ENV_NAME} python3 -c "import torch;y=torch.randn([3,5]);print(y);"; }
```

Comment:
Trailing "}" is an error here. Please also use the same smoke test script it should work for CPU